OpenEXR/exrcheck/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenEXR Project.
+
+add_executable(exrcheck main.cpp)
+target_link_libraries(exrcheck OpenEXR::IlmImf OpenEXR::IlmImfUtil)
+set_target_properties(exrcheck PROPERTIES
+  RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin"
+)
+# install(TARGETS exrcheck DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Comment:
Why not?

Reply:
This was discussed on the OpenEXR slack channel [(academysoftwarefdn#openexr)](https://academysoftwarefdn.slack.com/archives/CMLRW4N73) in February.

It was originally set to not install because it didn't seem appropriate to add a new binary install in a patch release. I overlooked adding it in at the next significant release.

`exrcheck` is largely useful when compiling the OpenEXR library with various sanitizer options, so it can detect programming errors in the library. Sanitizer builds are generally not installed anyway, so it doesn't seem useful to set it to install.

Do you have a case for it to be installed?

Reply:
Ok, good to know, thanks, yes I'm thinking to use it to check the exr file integrity, like when process crashed on writing or so, was confused a bit why it's documented but missed in destination.

Reply:
Yes `exrcheck` was also meant to be used that way, but it's not particularly efficient for that: currently it tries to read the file multiple times to exercise all the API. It needs a faster 'just validate file integrity' option that does the bare minimum to check the file is OK (including decompressing all the image data)